### PR TITLE
Added animation to the four squares changes

### PR DESCRIPTION
**Pull request by divyamayavan**

### CODE REVIEW

File: Week-2/2.2.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    #main {
+        position: relative;
+        left: 800px;
+        top: 300px;
+
+    }
+
+    #main:hover .square-red {
```
Comment (imsks) [inline Week-2/2.2.html:22]: Don't use ID when class can work.

File: Week-2/2.2.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    .square-yellow {
+        background-color: yellow;
+        height: 100px;
+        width: 100px;
+        transition-property: transform;
+        transition-duration: 1s;
+
+
+
+    }
```
Comment (imsks) [inline Week-2/2.2.html:58]: Remove empty lines

**Review by imsks — APPROVED** [review]

File: Week-2/2.2.html (hunk 1 above)
Comment (divyamayavan) [reply Week-2/2.2.html:22]: got it sir .I won't repeat this again

File: Week-2/2.2.html (hunk 2 above)
Comment (divyamayavan) [reply Week-2/2.2.html:58]: Okay

**Pull request merged**